api/apps/api/src/modules/scenario-specification/old/domain/computed-feature.ts

```diff
@@ -0,0 +1,8 @@
+export class ComputedFeature {
+  constructor(
+    public readonly name: string,
+    public readonly parentFeatureId: string,
+    public readonly derivedFromFeatureId: string,
+    public readonly id: string,
```

Comment:
looks a bit like a primitive obsession

Reply:
fixed in https://github.com/Vizzuality/marxan-cloud/pull/371/commits/4dbad00ade491ccca947032a295834751eab9b2d (removed draft code)